src/main/java/io/github/thebusybiscuit/slimefun4/utils/BlockUtils.java

```diff
@@ -0,0 +1,44 @@
+package io.github.thebusybiscuit.slimefun4.utils;
```

Comment:
Is this of any use to non-cargo related classes? If no, then it may be moved to the cargo package.
Otherwise it is fine to stay here.

Reply:
It looks like only cargo needed this, I think I will move it.